src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -228,11 +220,6 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
             // Fetching remote kernels cannot be done from cache.
             const promises = [
                 this.loadNotebookControllersImpl({
-                    listLocalNonPythonKernels: true,
-                    useCache: 'useCache'
-                }),
-                this.loadNotebookControllersImpl({
```

Comment:
Again, (same as earlier) we always retrieve both python & non-python kernel specs (hence no need to retrieve non-python kernel specs separately)